client/index.jsx

```diff
@@ -1,4 +1,48 @@
 import ReactDOM from 'react-dom';
 import React from 'react';
+import $ from 'jquery';
 
-ReactDOM.render(<div>'Hello World'</div>, document.getElementById('app'));
\ No newline at end of file
+class Reviews extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      reviews: []
+    };
+    this.get = this.get.bind(this);
```

Comment:
`this.get = this.get.bind(this);`

Should this be for this.getReviews? I don't see a 'get' method, but maybe I'm missing something?

Reply:
Thank you @garrowat! You're right, I was renaming things, good catch!